pyproject.toml

```diff
@@ -21,28 +21,29 @@ google-search-results = "*"
 pytest = "*"
 llama-index = "~0.9.0"
 duckduckgo-search = "*"
-crewai = {extras = ["tools"], version = "^0.22.5"}
+crewai = {extras = ["tools"], version = "^0.11.0"}
 # metagpt = "*" # Commented out because requires super old version of langchain, and conflicts with crewai.
 replicate = "*"
 typer = "^0.9.0"
 web3 = "^6.14.0"
 mypy = "^1.8.0"
-types-requests = "^2.31.0.20240106"
+types-requests = "^2.31.0.0"
 black = "^23.12.1"
 google-cloud-functions = "^1.16.0"
 google-cloud-resource-manager = "^1.12.0"
 poetry = "^1.7.1"
 poetry-plugin-export = "^1.6.0"
 functions-framework = "^3.5.0"
 cron-validator = "^1.0.8"
-prediction-market-agent-tooling = "^0.13.1"
+prediction-market-agent-tooling = {git = "https://github.com/gnosis/prediction-market-agent-tooling.git", rev = "evan/add-mech-client-dependency"}
 pydantic-settings = "^2.1.0"
 autoflake = "^2.2.1"
 isort = "^5.13.2"
 markdownify = "^0.11.6"
 tavily-python = "^0.3.1"
 # TODO remove when PR is merged
 microchain-python = {git = "https://github.com/evangriffiths/microchain.git", rev = "evan/token-tracker"}
+setuptools = "^69.5.1"
```

Comment:
Required by mech-client, although not a dependency. Will make a PR against the aea repo, but for now adding here

```
    from aea.crypto.plugin import load_all_plugins
  File "/Users/evan/dev/prediction-market-agent/.venv/lib/python3.10/site-packages/aea/crypto/plugin.py", line 26, in <module>
    from pkg_resources import EntryPoint, WorkingSet
ModuleNotFoundError: No module named 'pkg_resources'
```